src/adapter/input/post_router.py

```
@@ -35,91 +40,76 @@ def get_post_application_service():
+@router.post("",
             status_code=status.HTTP_201_CREATED,
-            responses={
-                403: {"description": "Authentication Code Error"},
-                500: {"description": "Internal Server Error"}
-            },
+            responses={403: {"description": "Authentication Code Error"},
+                    500: {"description": "Internal Server Error"}},
             response_model=PostCreateResponse)
-def create_post(post_create_request: PostCreateRequest, service: PostApplicationService = Depends(get_post_application_service)):
+@limiter.limit("5/minute") 
+def create_post(request: Request, post_create_request: PostCreateRequest, service: PostApplicationService = Depends(get_post_application_service)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)